pkg/test/harness.go

```diff
@@ -393,6 +402,16 @@ func (h *Harness) Stop() {
 		}
 	}
 
+	if h.bgProcesses != nil {
```

Comment:
Fun fact, I've asked for a test for this case in the original #23 PR where this [bug was introduced](https://github.com/kudobuilder/kuttl/pull/23#discussion_r395609070) ;)

Reply:
@zen-dog definitely!   added!